src/Compilers/Core/Portable/Symbols/ISymbol.cs

```diff
@@ -203,7 +203,7 @@ public interface ISymbol : IEquatable<ISymbol?>
         ISymbol OriginalDefinition { get; }
 
         void Accept(SymbolVisitor visitor);
-        TResult Accept<TResult>(SymbolVisitor<TResult> visitor);
+        TResult? Accept<TResult>(SymbolVisitor<TResult> visitor);
```

Comment:
We may want to test insert this change before merging to roslyn master, so that we can prepare fixes for any VS-side build breaks.

Reply:
Should be okay (I've run a test insertion on previous PR with similar effect), but it's good to be sure. Started https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/results?buildId=4276225&view=results

Reply:
Succeeded. I'll go ahead and merge